src/main/java/su/litvak/chromecast/api/v2/ChromeCast.java

```diff
@@ -86,6 +86,21 @@ public String getApplication() {
     public void setApplication(String application) {
         this.application = application;
     }
+    
+    /** Returns the {@link #channel} and opens it if it's not yet or
+     * no longer open.
+     * @return an open channel.
+     */
+    private Channel channel() throws IOException {
```

Comment:
I had another commit making the whole thing an [java.io.AutoClosable](https://docs.oracle.com/javase/7/docs/api/java/lang/AutoCloseable.html), but this really breaks the API logic because `close()` is not `disconnect()` and keeping both will confuse everyone.
But it fits very good to auto connect.

Explicit API usage is for people reading the API docs. Yep, the docs need a friendly hand..
At work we say that a good API doesn't need many docs and has a joy of use. 
I :heart:  APIs that are impossible to misuse :+1: 

Let's break things, it's a 0.X version :rofl: 
Maybe keep this PR until the first 1.X version?
Curious about your thoughts!